config/rewrites.d/developer.rackspace.com.json

```diff
@@ -322,14 +322,14 @@
         {
           "description": "Redirect unversioned RPC URL to current version",
           "from": "^\\/docs\\/private-cloud\\/rpc\\/?$",
-          "to": "/docs/private-cloud/rpc/v14",
+          "to": "/docs/private-cloud/rpc/v16",
           "rewrite:": false,
           "status": 301
         },
         {
-          "description": "Redirect older RPC URLs to current version",
+          "description": "Redirect older RPC URLs to current version minus one",
```

Comment:
Just a nit, but this should say `Redirect retired RPC versions to current version`